includes/class-stats.php

```diff
@@ -3149,9 +3023,85 @@ private function set_date_ranges() {
 					break;
 			}
 
-			$dates['range'] = $range;
+			if ( ! empty( $dates ) ) {
+				// Convert the values to the UTC equivalent so that we can query the database using UTC.
+				$dates['start'] = edd_get_utc_equivalent_date( $dates['start'] );
+				$dates['end']   = edd_get_utc_equivalent_date( $dates['end'] );
+				$dates['range'] = $range;
 
-			$this->relative_date_ranges[ $range ] = $dates;
+				$this->relative_date_ranges[ $range ] = $dates;
+			}
 		}
+
+	}
+
+	/**
+	 * Based on the query_vars['revenue_type'], use gross or net statuses.
+	 *
+	 * @since 3.0
+	 *
+	 * @return array The statuses of orders to use for the stats generation.
+	 */
+	private function get_revenue_type_statuses() {
+		if ( 'net' === $this->query_vars['revenue_type'] ) {
+			return edd_get_net_order_statuses();
+		}
+
+		return edd_get_gross_order_statuses();
+	}
+
+	/**
+	 * Based on the query_vars['revenue_type'], use just sale or also include refunds.
+	 *
+	 * @since 3.0
+	 *
+	 * @return array The order types to use when generating stats.
+	 */
+	private function get_revenue_type_order_types() {
+		$order_types = array( 'sale' );
+		if ( 'net' === $this->query_vars['revenue_type'] ) {
+			$order_types[] = 'refund';
+		}
+
+		return $order_types;
+	}
+
+	/**
+	 * Generates output for the report tiles when a relative % change is requested.
+	 *
+	 * @since 3.0
+	 *
+	 * @param int|float $total     The primary value result for the stat.
+	 * @param int|float $relative  The value relative to the previous date range.
+	 * @param bool      $reverse   If the stat being displayed is a 'reverse' state, where lower is better.
+	 */
+	private function generate_relative_markup( $total = 0, $relative = 0, $reverse = false ) {
```

Comment:
The `$reverse` parameter is not being used in this function, although at least the `get_order_refund_amount` defines it.

I'm also seeing inconsistencies in whether the relative markup is displaying...sometimes it is, sometimes not, on the same tiles. Here's "last quarter":
![image](https://user-images.githubusercontent.com/4853066/176676192-b6cd3355-928a-49df-9408-ce887d386e3b.png)

And here is "this quarter":
![image](https://user-images.githubusercontent.com/4853066/176676319-90c7d0fd-7953-4229-b79e-28b58c27567b.png)